### PR TITLE
first-working-version

### PR DESCRIPTION
**Pull request by ishaimika**
First working version of the PayFunc CLI.
Made it possible to cancel, charge, create, list and verify orders.
Added merchant module with commands create, list and update.

### CODE REVIEW

**Comment by ishaimika** [comment]
Added remove server, charge order, refund order and cancel order.

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        ]
+      },
+      {
+        "type": "npm",
+        "script": "test",
+        "problemMatcher": []
+      }
+    ]
+  }
+
```
[the remark this answers is not in the document]
Comment (ishaimika) [reply .vscode/tasks.json]: Fixed.

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+			"skipFiles": [
+				"<node_internals>/**"
+			],
+			"program": "${workspaceFolder}/dist/index.js",
+			"preLaunchTask": "build",
+			"outFiles": [
+				"${workspaceFolder}/dist/**/*.js"
+			]
+		},
+		{
```
[the remark this answers is not in the document]
Comment (ishaimika) [reply .vscode/launch.json]: Removed.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+			"type": "node",
+			"request": "launch",
+			"name": "help",
+			"args": [
+				"help",
+				"authorization"
```
[the remark this answers is not in the document]
Comment (ishaimika) [reply .vscode/launch.json]: Line removed.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+ "scripts": {
+  "dev": "watch 'npm run build' .",
+  "lint": "tslint --config tslint.json --project .",
+  "build": "tsc -p .",
+  "test": "node dist/index.js --server env test",
+  "prepare": "npm run build",
+  "clean": "rm -rf dist node_modules coverage"
+ },
+ "dependencies": {
+  "@cardfunc/model": "^0.0.54",
```
Comment (simonmika) [inline package.json:56]: Missing dependency on `@payfunc/model`

File: package.json [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  "@types/dotenv": "^8.2.0",
+  "authly": "0.0.39",
+  "dotenv": "^8.2.0",
+  "gracely": "0.0.31",
+  "isoly": "0.0.13",
+  "node-fetch": "^2.6.0",
+  "node-persist": "^3.0.5",
+  "open": "^7.0.4"
```
Comment (simonmika) [inline package.json:64]: Do we use this?

File: package.json [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+ },
+ "devDependencies": {
+  "@types/jest": "^25.1.2",
+  "@types/node": "^13.7.0",
+  "@types/node-fetch": "^2.5.7",
+  "@types/node-persist": "^3.0.0",
+  "@types/open": "^6.2.1",
```
Comment (simonmika) [inline package.json:71]: Don't forget to remove this if the above is not needed.

File: utility/postForm.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import { default as fetch } from "node-fetch"
+import * as querystring from "querystring"
+export async function postForm(url: string, data: { [field: string]: string }): Promise<{ [field: string]: string }> {
```
Comment (simonmika) [inline utility/postForm.ts:3]: Is this really used?

**Pull request merged**